main.py

```diff
@@ -29,51 +66,19 @@ def codebreaker():
             x = int(input('How many guesses would you like to attempt?: '))
         except ValueError:
             print(f'Invalid input, use an integer greater than 0')
-    while y < 1 or y > 72:
+    while y not in range(1, 72):
         try:
             y = int(input('How long do you want the code? (From 1 to 72): '))
         except ValueError:
             print(f'Invalid input, use an integer from 1 to 72')
-    for i in range(y):
-        code.append(box.pop(0))
```

Comment:
rather than looping over this. We can use the list slicing [index start: index end]
see: https://github.com/DaneRosa/dr-ml/pull/2/files#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R74